internal/k8s/watch.go

```diff
@@ -397,9 +396,12 @@ func watchEverythingLoop(ctx context.Context, ch chan<- watch.Event, watchers []
 			//    https://github.com/windmilleng/tilt/pull/1647. I feel like we would have noticed this with pods,
 			//    though.
 			// 4. Maybe setting up 100+ watches taxes the system and causes it to drop connections
-			logger.Get(ctx).Infof("DEBUG: ok was false for %v", gvrs[chosen])
-			cleanUp(ch, watchers)
-			return
+			logger.Get(ctx).Infof("Failed watching K8S objects of type %v. This shouldn't happen. Please report a bug at: https://github.com/windmilleng/tilt/issues", gvrs[chosen])
+			analytics.Get(ctx).Incr("dynamic.watcher.closed", map[string]string{})
```

Comment:
it would! But that's user data, so I figured we'd leave it off out of privacy concerns, and leave it to users to include that info in the bug report. I don't know if that's overly cautious.